nextjs-13/src/app/api/v1/newsletter/[id]/route.ts

```diff
@@ -31,3 +34,43 @@ export async function DELETE(
     }
   });
 }
+
+export async function PUT(
+  req: NextRequest,
+  { params }: { params: { id: string } }
+) {
+  return appHandler(req, async (currentUser) => {
+    try {
+      const { name, content } = await req.json();
+
+      const data = { name, content };
+
+      const where = {
+        id: params.id,
+        user: currentUser,
+      };
+
+      const result = await updateNewsletter({
+        where,
+        data,
+      });
+
+      if (result.count === 0) {
```

Comment:
This could be simplified to:

```suggestion
      if (!result.count) {
```

Reply:
Improved [here](https://github.com/nimblehq/js-framework-benchmark/pull/117/commits/bfb0be0590523740e6bf06ff7594d02cc26624a1#diff-e2c82a3d6cd014d8235827328f63df5db8efb48bbd67175aad18498f1950b96dR58)